components/calendar/calendar.js

```diff
@@ -93,13 +93,6 @@ export function getDatesInMonthArray(shownMonth, shownYear) {
 	return dates;
 }
 
-export function getDisabled(date, min, max) {
```

Comment:
The change to calendar was just to move this functionality to `dateTime` helper since input-date now needs it.